pkg/cmd/triggertemplate/describe.go

```diff
@@ -61,11 +60,19 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .TriggerTemplate.Name }
 {{- end }}
 {{- end }}
 
-{{decorate "resources" ""}}{{decorate "underline bold" "ResourceTemplates\n"}}
+{{- $errValue := checkError .TriggerTemplate.Spec.ResourceTemplates }}
+{{- if ne $errValue "" }}
+
+ {{ $errValue }}
+{{- else }}
 
-{{- $value := getResourceTemplate .TriggerTemplate }}
-{{-  if ne $value "" }}
-{{ $value }}
+{{decorate "resources" ""}}{{decorate "underline bold" "ResourceTemplates\n"}}
+ NAME	GENERATENAME	KIND	APIVERSION
+{{- range $p := .TriggerTemplate.Spec.ResourceTemplates }}
+{{- $value := getResourceTemplate $p }}
+ {{- $name := format $value.GetName }}
```

Comment:
can't we put this in the next line? Are there some issues with that?

Reply:
`decorate` doesn't take more than 2 argument so modified that way

Reply:
I gave it a try today, this works

```
{{- range $p := .TriggerTemplate.Spec.ResourceTemplates }}
{{- $value := getResourceTemplate $p }}
 {{ format $value.GetName | decorate "bullet" }}	{{ format $value.GetGenerateName }}	{{ $value.GetKind }}	{{ $value.GetAPIVersion }}
{{- end }}
```

Reply:
Thank you 
Updated